NEWS

```diff
@@ -1,3 +1,18 @@
+Flocker Next Release
```

Comment:
I'd put the date here. This will be helpful for the next release engineer so they can look only at the `git log` since today's date.